html/about.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/3da71c9faf.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../style/main.css">
+    <link rel="stylesheet" href="../style/about.css">
+    <script src="../javascript/main.js" defer></script>
+    <title>Global Drummers Workshop</title>
+</head>
+
+<body class="lato-font">
+    <header id="nav-container">
+        <a class="logo-container" href="../index.html">
+            <img class="logo" src="../assets/icons/Drum-festival-logo.svg" alt="Drum Festival Logo">
+        </a>
+        <nav class="navbar">
+            <ul class="navlinks-wrapper">
+                <li>
+                    <a href="./about.html">About</a>
+                </li>
+                <li>
+                    <a href="../index.html#programs">Program</a>
+                </li>
+                <li>
+                    <a href="#Sponsors">Sponsor</a>
+                </li>
+                <li>
+                    <a href="../index.html#performing-artist">News</a>
+                </li>
+            </ul>
+
+            <button type="button" class="nav-join-btn">
+                Join Drum Festival
+            </button>
+
+            <div class="hamburger-btn">
+                <img class="hamburger" src="../assets/icons/menu-icon.png" alt="hamburger icon">
+            </div>
+        </nav>
+    </header>
+
+    <section id="about-headline-container">
+        <div class="orange-red lato-font">"Hello! Drummers!!"</div>
+        <h1 class="orange-red cocogoose-font">GLOBAL DRUM FESTIVAL & WORKSHOP 2022</h1>
+        <p class="black-like lato-font about-headline-body">Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+            Nisi corporis ipsam totam enim accusantium vero culpa, harum magni laudantium ratione ullam quam, esse sequi
+            inventore libero soluta unde facilis quod at veritatis! Aut, impedit minima!</p>
+
+        <p class="black-like lato-font">Please do reach reach out to us via email for any further questions about this
+            event and how you can participate
+            <br>
+            <span>
+                <a class="contact-us" href="#">
+                    globaldrumfestival@gdf-portal.org
+                </a>
+            </span>
+        </p>
+
+    </section>
+    <section id="past-global-drum-fest">
```

Comment:
- [ ] Kindly check the section highlighted below, your design is missing one section.
<img width="557" alt="image" src="https://user-images.githubusercontent.com/61408860/169386634-40ea8a33-0b9c-4325-bda3-9b173fac8434.png">

Reply:
- [ ] This issue is well resolved :heavy_check_mark: